src/node_process_methods.cc

```diff
@@ -650,6 +715,7 @@ static void CreatePerIsolateProperties(IsolateData* isolate_data,
   SetMethod(isolate, target, "availableMemory", GetAvailableMemory);
   SetMethod(isolate, target, "rss", Rss);
   SetMethod(isolate, target, "cpuUsage", CPUUsage);
+  SetMethod(isolate, target, "threadCpuUsage", ThreadCPUUsage);
```

Comment:
Could you also update the typings folder with the correct type? That way, we’ll have even better intellisense support!

Reply:
I will.